Jenkinsfile

```diff
@@ -91,6 +91,15 @@ def deploy(deploy_environment) {
       sh('git fetch')
       sh('git checkout stable')
 
+      withAWS(credentials: 'jenkins-read-wordlist-credentials') {
+        s3Download(
+          file:'tmp/wordlist',
```

Comment:
I note previously in this file spaces are used between named arguments and the values :-/